notebooks/official/generative_ai/vertex_sdk_llm_snippets.ipynb

```diff
@@ -0,0 +1,687 @@
+{
```

Comment:
<p>The fonts of the heading doesn't seem right. Does this comply with our template?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>nope</p>